site/weavedns.md

```diff
@@ -254,60 +232,13 @@ can force weaveDNS to use a different domain by launching it
 with the `--domain` argument. For example,
 
 ```bash
-$ weave launch-dns 10.2.254.1/24 --domain="mycompany.local."
+$ weave launch-dns --domain="mycompany.local."
 ```
 
 The local domain should end with `local.`, since these names are
 link-local as per [RFC6762](https://tools.ietf.org/html/rfc6762),
 (though this is not strictly necessary).
 
-## <a name="without-run"></a>Using weaveDNS without `weave run`
```

Comment:
The link from the ToC is still there. Also check there isn't a reference to it from elsewhere.